distribution/src/bin/elasticsearch-env-from-file

```diff
@@ -20,19 +20,43 @@ for VAR_NAME_FILE in $(env | cut -f1 -d= | grep '_FILE$'); do
     fi
 
     if [[ ! -e "${!VAR_NAME_FILE}" ]]; then
-      echo "ERROR: File ${!VAR_NAME_FILE} from $VAR_NAME_FILE does not exist" >&2
+      # Maybe the file doesn't exist, maybe we just can't read it due to file permissions.
+      # Check permissions on each part of the path
+      if ! echo "${!VAR_NAME_FILE}" | grep -q '^/'; then
+        path=.
```

Comment:
If this conditional doesn't execute what is the value of `path`? AFAICS we rely on some `$path` value in the following `while` loop (lines 29-34)

Reply:
The value is empty. I've added an explicit initialisation for this.